src/Avalonia.Controls/DataGrid/DataGridBoundColumn.cs

```diff
@@ -0,0 +1,313 @@
+                        //_binding.ValidatesOnDataErrors = true;
+                        //_binding.ValidatesOnExceptions = true;
+                        //_binding.NotifyOnValidationError = true;
+                        //_binding.UpdateSourceTrigger = UpdateSourceTrigger.Explicit;
+
+                        // Apply the new Binding to existing rows in the DataGrid
+                        if (OwningGrid != null)
+                        {
+                            OwningGrid.OnColumnBindingChanged(this);
+                        }
+                    }
+
+                    RemoveEditingElement();
+                }
+            }
+        }
+
+        /// <summary>
+        /// The binding that will be used to get or set cell content for the clipboard.
+        /// If the base ClipboardContentBinding is not explicitly set, this will return the value of Binding.
+        /// </summary>
+        public override IBinding ClipboardContentBinding
+        {
+            get
+            {
+                return base.ClipboardContentBinding ?? Binding;
+            }
+            set
+            {
+                base.ClipboardContentBinding = value;
+            }
+        }
+
+        //TODO Rename
+        //TODO Validation
+        protected sealed override IControl GenerateEditingElement(DataGridCell cell, object dataItem, out ICellEditBinding editBinding)
+        {
+            IControl element = GenerateEditingElementDirect(cell, dataItem);
+            editBinding = null;
+
+            if (Binding != null)
+            //if (Binding != null || !DesignerProperties.IsInDesignTool)
+            {
+                //var t1 = Binding.Initiate(textBox, BindingTarget, anchor: null, enableDataValidation: true);
+                //BindingOperations.Apply(textBox, BindingTarget, t1, null);
+
+                editBinding = BindEditingElement(element, BindingTarget, Binding);
+                //element.Bind(BindingTarget, Binding);
+                //textBox.SetBinding(BindingTarget, Binding);
+            }
+
+            return element;
+        }
+
+        private static ICellEditBinding BindEditingElement(IAvaloniaObject target, AvaloniaProperty property, IBinding binding)
+        {
+            var result = binding.Initiate(target, property, enableDataValidation: true);
+
+            if (result != null)
+            {
+                if(result.Subject != null)
+                {
+                    var bindingHelper = new CellEditBinding(result.Subject);
+                    var instanceBinding = new InstancedBinding(bindingHelper.InternalSubject, result.Mode, result.Priority);
+
+                    BindingOperations.Apply(target, property, instanceBinding, null);
+                    return bindingHelper;
+                }
+
+                BindingOperations.Apply(target, property, result, null);
+            }
+
+            return null;
+        }
+        
+        /*
```

Comment:
This file has several commented blocks. Is it intended?

Reply:
Resolved with commit 1d9593a